.golangci.yml

```diff
@@ -15,15 +15,13 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-#
 
 # options for analysis running
 run:
-  go: '1.21'
```

Comment:
If we remove this line , which golang version will be used ?

Reply:
I believe it will use the default version executed with 'go'.

Reply:
> If we remove this line , which golang version will be used ?

Default: use Go version from the go.mod file

Quoted from golangci-lint configuration
```
  # Define the Go version limit.
  # Mainly related to generics support since go1.18.
  # Default: use Go version from the go.mod file, fallback on the env var `GOVERSION`, fallback on 1.18
  go: '1.19'
```

Reply:
While go.mod still use version 1.20. Should we upgrade it to 1.21 ? gentle ping @craigcondit, @FrankYang0529 . If so, I'm willing to do this.

Reply:
No, we should leave it at 1.20 as that impacts downstream consumers. We don't (yet) require any go 1.21 features to compile.